tests/Integration/Behaviour/Features/Scenario/Product/add_virtual_product_file.feature

```diff
@@ -0,0 +1,93 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags virtual-product-file
+@reset-database-before-feature
+@virtual-product-file
+@reset-downloads-after-feature
+Feature: Add virtual product file from BO (Back Office).
+  As an employee I want to be able to add file of virtual product.
+
+  Scenario: I add virtual product file without limiting access days, download times and without expiration date
+    And I add product "product1" with following information:
```

Comment:
```suggestion
    When I add product "product1" with following information:
```

Reply:
it should be Given*. 
Fixed :heavy_check_mark: